src/features/User/Ui/Footer/Footer.tsx

```diff
@@ -1,9 +1,113 @@
-import React from 'react'
+import * as React from "react";
+import { Container, Grid, Box,ScopedCssBaseline } from "@mui/material";
+import MenuItem from "@mui/material/MenuItem";
+import Stack from "@mui/material/Stack";
+import { Typography, List, ListItem } from "@mui/material";
+import Link from "@mui/material/Link";
 
 const Footer = () => {
+  const footerSections = [
+    {
+      title: "For Beginners",
+      links: [
+        { text: "New Account" },
+        { text: "Start Booking a Room" },
+        { text: "Use Payments" },
+      ],
+    },
+    {
+      title: "Explore Us",
+      links: [
+        { text: "Our Careers" },
+        { text: "Privacy" },
+        { text: "Terms & Conditions" },
+      ],
+    },
+    {
+      title: "Connect Us",
+      contactInfo: [
+        { text: "support@staycation.id" },
+        { text: "021 - 2208 - 1996" },
+        { text: "Staycation, Jakarta" },
+      ],
+    },
+  ];
+
   return (
-    <div>Footer</div>
-  )
-}
+    <ScopedCssBaseline  sx={{ backgroundColor: "#f5f5f5" }}>
+    <Container component="footer">
```

Comment:
Footer elements are not aligned at top
Also lists in the footer are not aligned with their corresponding headers

![image](https://github.com/SAMIYAghb/hotel-system/assets/64327685/7ea76274-753f-4daa-8fdf-1650a823e45d)

Reply:
done